myScript.js

```diff
@@ -0,0 +1,81 @@
+function sortUserNames() //Task 1
+{
+    const userNames = ["Петрик Ольга Іванівна", "Гнатюк Петро Антонович", "Рудко Андрій Опанасович"];
+    let initials;
+
+    initials = new Array();
```

Comment:
```suggestion
    let initials = [];
```
Можна зробити простіше